<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QMainWindow" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Register</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font-family: century gothic;
}

QWidget{
background-image: url(:/images/images/background_register.png);
}

QFrame#contenedor{
background:#1d1d1d;
border-radius:15px;
}

QToolButton{
background:#2d89ef;
border-radius:15px;
}

QLabel#registro{
color:white;
font-size:24px;
}

QLabel{
font-weight:bold;
color:white;
}

QLineEdit{
background:transparent;
border:none;
border-bottom:1px solid white;
color:white;
}

QPushButton{
background:#2d89ef;
color:white;
border-radius:15px;
}

QPushButton:hover{
background:#2b5797;
color:white;
border-radius:15px;
font-weight: bold;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="contenedor">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>371</width>
      <height>611</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QToolButton" name="toolButton">
     <property name="geometry">
      <rect>
       <x>164</x>
       <y>20</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/image.login.png</normaloff>images/image.login.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>REGISTRO</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>151</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombre/s</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNombre">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>321</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>16</number>
     </property>
     <property name="placeholderText">
      <string>Nombre/s</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>250</y>
       <width>151</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Apellido/s</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtApellido">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>280</y>
       <width>321</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>16</number>
     </property>
     <property name="placeholderText">
      <string>Apellido/s</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>330</y>
       <width>151</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>E-mail</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtEmail">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>360</y>
       <width>321</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>25</number>
     </property>
     <property name="placeholderText">
      <string>example@mail.com</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>410</y>
       <width>151</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Contraseña</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtContra">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>440</y>
       <width>321</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>12</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>•••••••••••</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCuenta">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>530</y>
       <width>321</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>CREAR CUENTA</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
